src/data.js

```diff
@@ -39,12 +39,9 @@ export const getAllData = async (list, all = false) => {
   const localData = localStorageHelper('get');
   const data = {};
 
-  if (localData && localData.date) {
```

Comment:
No need to test `localData` if `localData.date` is tested

Reply:
if localData is null localData.date will throw an error. So you do need this

Reply:
you could change it to localData?.date (i think this project allows optional chaining)

Reply:
Corrected 👍